e2e/atlas/logs_test.go

```diff
@@ -48,6 +48,7 @@ func TestLogs(t *testing.T) {
 	logTypes := []string{
 		"mongodb.gz",
 		"mongos.gz",
+		"mongosqld.gz",
```

Comment:
> i haven't made e2e changes part of this PR

@shalka this change is the one I refer to, this will fail once it gets merged, right?

Reply:
You are exactly right! This was an oversight on my part and I've pushed a correction.